docs/websites/cms/drupal/how-to-update-drupal-8-on-centos-8/index.md

```diff
@@ -0,0 +1,142 @@
+---
+author:
+    name: Linode
+    email: docs@linode.com
+description: 'This guide will show you how to update your Drupal 8 installation running on a CentOS 8 Linode.'
+og_description: 'This guide will show you how to update your Drupal 8 installation running on a CentOS 8 Linode.'
+keywords: ["cms", "apache", "php", "content management system", "drupal 8", "update"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-02-21
+modified_by:
+    name: Linode
+published: 2020-02-21
+title: How to Update Drupal 8 on CentOS 8
+h1_title: Update Drupal 8 on Centos 8
+---
+
+Drupal 8 is the latest version of the popular [Drupal](https://www.drupal.org/) content management system. While Drupal 8.1 includes a simple feature for incremental updates, you must preform manual Drupal core updates for any preceding versions. This guide demonstrates how to manually install an incremental Drupal 8 update on your Linode. The examples in this guide assume you have a functional [Drupal 8 installation](/docs/websites/cms/drupal/how-to-install-and-configure-drupal-on-centos-8/) running a [LAMP stack](/docs/web-servers/lamp/how-to-install-a-lamp-stack-on-centos-8/) on CentOS 8.
+
+## Before You Begin
+
+1. Complete all the steps in the [Install and Configure Drupal 8 on CentOS 8](/docs/websites/cms/drupal/how-to-install-and-configure-drupal-on-centos-8/) guide.
+
+1. If you followed the [Install and Configure Drupal 8 on CentOS 8](/docs/websites/cms/drupal/how-to-install-and-configure-drupal-on-centos-8/) guide, your site's document root should be in the `/var/www/html/example.com/` directory, where `example.com` is your own site's domain name. You can list all your directories in `/var/www/html` to verify the location of your site's document root.
+
+        ls /var/wwww/html/
+
+1. Update your CentOS 8 system if you did not complete that step while installing Drupal 8.
+
+        sudo yum update
+
+    {{< content "limited-user-note-shortguide" >}}
+
+## Create Backups
+
+In this section, you will create an archive of your Drupal site's files and store the archive in a `backups` directory. If needed, you could extract the compressed files in your backup archive to restore the state of your site prior to updating it.
+
+1. Create a `backups` directory in your site's document root to store any backups you make of your Drupal site.
+
+        sudo mkdir /var/www/html/example.com/backups
+
+1. Create an archive of your existing site files and move it into the `backups` directory. Ensure you replace `example.com` with your own site's domain name.
+
+        cd /var/www/html/example.com/public_html
+        sudo tar -cvzf example.com-BCKP-$(date +%Y%m%d).tar.gz ./
+        sudo mv -v example.com-BCKP-*.tar.gz ../backups
+
+    {{< note >}}
+This process can also be scripted and run on a regular basis using [cron](/docs/tools-reference/tools/schedule-tasks-with-cron/).
+    {{</ note >}}
```

Comment:
You can also schedule backups with the 'backup_migrate' module in the Drupal UI as well.

Reply:
Looks like that module has a number of outstanding bugs, I think for this guide we can leave it out.